tokens/connect/polkadotjs.md

```diff
@@ -48,24 +48,24 @@ description: 根据此教程快速学习如何使用Polkadot.js Apps与Moonbeam
 
 ![Connect to Moonbase Alpha](/images/tokens/connect/polkadotjs/polkadotjs-app-5.png)
 
-点击“Next”，设置账户名和密码后关闭设置向导的弹窗。收到确认消息后，您会在Accounts标签中看到相应地址和余额（在本示例中为Bob的地址）。除此之外，我们还可以覆盖MetaMask扩展，可以看到两个余额是相同的。
+点击**Next**，设置账户名和密码后关闭设置向导的弹窗。收到确认消息后，您会在**Accounts**标签中看到相应地址和余额（在本示例中为Bob的地址）。除此之外，您还可以覆盖MetaMask扩展，可以看到两个余额是相同的。
 
 ![Connect to Moonbase Alpha](/images/tokens/connect/polkadotjs/polkadotjs-app-6.png)
 
 ## 通过Substrate API发送交易 {: #sending-a-transaction-through-substrates-api } 
 
-现在，我们将演示Moonbeam Unified Accounts方案。通过Substrate API使用Polkadot.js App来创建交易。请注意，我们使用的是以太坊式H160地址来和Substrate交互。为此，我们导入了另一个名为Charley的账户，该账户中有5枚`DEV`代币。
+现在，您将演示Moonbeam Unified Accounts方案。通过Substrate API使用Polkadot.js App来创建交易。请注意，您使用的是以太坊式H160地址来和Substrate交互。为此，您导入了另一个名为Charley的账户，该账户中有5枚`DEV`代币。
 
 ![Connect to Moonbase Alpha](/images/tokens/connect/polkadotjs/polkadotjs-app-7.png)
 
-下一步，点击“send”按钮，随后会出现一个弹窗，指引您按照操作发送交易。设置发送地址和金额，在本示例中为5个DEV代币。一切就绪后，就可以点击“Make Transfer”按钮。
+下一步，点击**Send**按钮，随后会出现一个弹窗，指引您按照操作发送交易。设置发送地址和金额，在本示例中为5个DEV代币。一切就绪后，就可以点击**Make Transfer**按钮。
```

Comment:
they don't actually click on the word Send here, it's a send button... so this doesn't need to be bold and it can be translated